examples/demo-auth/atmos.yaml

```
@@ -0,0 +1,83 @@
+      profile: "acme-identity"
+      role_name: "IdentityManagersTeamAccess"
+      account_name: "core-identity"
+    idp-superuser:
+      type: aws/user
+
+  identities:
+    # Used in tests
+    oidc:
+      provider: oidc
+      default: false # we use the identity flag in the test custom command
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)